R/pkg/NAMESPACE

```diff
@@ -6,10 +6,15 @@ importFrom(methods, setGeneric, setMethod, setOldClass)
 #useDynLib(SparkR, stringHashCode)
 
 # S3 methods exported
+export("sparkR.session.getOrCreate")
```

Comment:
while SparkSession is the main entry, SparkContext does remain. My thoughts are:
1. Keep sparkR.init() as is. It still returns the SparkContext.
2. Add new API like SparkRSession.init(), and sparkRSession.stop(). 
   SparkRSession.init() has two forms:
   A. it can accept a SparkContext as a parameter, and no other Spark Configurations.
   B. Just like the current sparkR.session.getOrCreate(), it internally creates SparkContext.
3. Keep sparkRSQL.init() and sparkRHive.init() for backward compatibility, while they are updated to call SparkRSession.init().